DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h

```diff
@@ -87,9 +87,10 @@ class DetID
   static constexpr ID RCH = 23;
   static constexpr ID MI3 = 24;
   static constexpr ID ECL = 25;
-  static constexpr ID Last = ECL;
+  static constexpr ID FVD = 26;
+  static constexpr ID Last = FVD;
 #else
-  static constexpr ID Last = FOC; ///< if extra detectors added, update this !!!
+  static constexpr ID Last = FVD; ///< if extra detectors added, update this !!!
```

Comment:
This should be the last det. of the non-upgraded list, keep FOC here.

Reply:
Corrected